AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5763,7 +5762,6 @@ proshop.de#%#document.cookie = "CookieInformationConsent=%7B%22consents_approved
 raiba-ke-oa.de#%#//scriptlet("set-constant", "Object.prototype.cookieConsentState", "true")
 teb.com.tr#%#//scriptlet('remove-class', 'ckPp', 'body')
 new-work.se,xing.com#%#document.cookie="userConsent=%7B%22marketing%22%3Afalse%2C%22version%22%3A%221%22%7D; path=/;";
-wetter.com#%#AG_onLoad(function(){if("undefined"!=typeof localStorage)try{if(!localStorage.getItem("uc_tcf")){var b=new MutationObserver(function(){var a=document.querySelector("button#cmp-btn-accept");a&&setTimeout(function(){a.click();b.disconnect()},1E3)});b.observe(document,{childList:!0,subtree:!0});setTimeout(function(){b.disconnect()},1E4)}}catch(a){}});
```

Comment:
Hmmm, it seems that this rule still works fine on my end.
@BlazDT Could you please take a look?

Reply:
I checked again and it works with Safari extension, and doesn't work with Chrome, Firefox extensions and the program

Reply:
@AdamWr It is not triggered if `uc_tcf` exists in localStorage

Reply:
If I'm not wrong, it shouldn't exists until cookie notice is closed/accepted.
By the way, it still works fine on my end with AdGuard for Windows in Chrome/Firefox.

Reply:
On my end also not working. I do not clear history in general, and visited this site a long time ago. Maybe uc_tcf was saved.